torchvision/csrc/cpu/video_reader/VideoReader.cpp

```diff
@@ -0,0 +1,370 @@
+#include "VideoReader.h"
+#include <ATen/ATen.h>
+#include <c10/util/Logging.h>
+#include <exception>
+#include "FfmpegDecoder.h"
+#include "FfmpegHeaders.h"
+#include "util.h"
+
+using namespace std;
```

Comment:
Linker is failing on Windows, and I think this can be solved in the same way as in https://github.com/pytorch/vision/pull/1320/commits/8bc85b57f8a454f38c4e069c00f0623e20775e65